default.nix

```diff
@@ -1,36 +1,40 @@
 let
   pkgs = import <nixpkgs> {};
-  easy-dhall = import ./easy-dhall.nix { inherit pkgs; };
-
+  easy-dhall = import ./easy-dhall.nix {
+    inherit pkgs;
+  };
   inputs = rec {
-    purs = import ./purs.nix { inherit pkgs; };
+    purs = import ./purs.nix {
+      inherit pkgs;
+    };
     purs-simple = purs;
     purescript = purs;
-
-    psc-package-simple = import ./psc-package-simple.nix { inherit pkgs; };
+    psc-package-simple = import ./psc-package-simple.nix {
+      inherit pkgs;
+    };
     psc-package = psc-package-simple;
-
-    purp = import ./purp.nix { inherit pkgs; };
-
-    inherit (easy-dhall)
-      dhall-simple
-      dhall-json-simple;
-    spacchetti-cli = import ./spacchetti-cli.nix { inherit pkgs; };
-
-    spago = import ./spago.nix { inherit pkgs; };
-
-    psc-package2nix = import ./psc-package2nix.nix { inherit pkgs; };
-
-    zephyr = import ./zephyr.nix { inherit pkgs; };
+    purp = import ./purp.nix {
+      inherit pkgs;
+    };
+    inherit (easy-dhall) dhall-simple dhall-json-simple;
+    spacchetti-cli = import ./spacchetti-cli.nix {
+      inherit pkgs;
+    };
+    spago = import ./spago.nix {
+      inherit pkgs;
+    };
+    psc-package2nix = import ./psc-package2nix.nix {
+      inherit pkgs;
+    };
+    zephyr = import ./zephyr.nix {
+      inherit pkgs;
+    };
   };
-
   buildInputs = builtins.attrValues inputs;
 in inputs // {
   inputs = inputs;
-
   buildInputs = buildInputs;
-
   shell = pkgs.runCommand "easy-purescript-nix-shell" {
     buildInputs = buildInputs;
   } "";
-}
+}
```

Comment:
I'm not sure about smushing all the lines together or removing the final newline. Maybe collapsing multiple blank lines to just one?

Reply:
yeah, it seems like i need to just adjust the `pretty` function to add one final newline in the formatter, and otherwise i might need to just think about how to intercalate just two newlines between each bind or something